tests/jeos/glibc_locale.pm

```diff
@@ -0,0 +1,46 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: This module installs glibc-locale and sets the locale to the one mentioned in the test variable.
+# Maintainer: Ciprian Cret <ccret@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils;
+use version_utils qw(is_opensuse is_sle is_jeos);
+
+sub run {
+    zypper_call('in glibc-locale');
+    my $locale = get_var('JEOSINSTLANG');
+    my $output = script_output("localectl list-locales | grep $locale.utf8");
+
+    die "Test locale not found in the available ones" unless ($output =~ $locale);
+
+    assert_script_run("localectl set-locale LANG=$locale.UTF-8");
+    assert_script_run("export LC_ALL='$locale.UTF-8'");
+
+    my (%args) = @_;
+    my $user = $args{user_is_root};
+    my $lang = get_required_var('JEOSINSTLANG', 'en_US');
```

Comment:
Requiring the test variable and providing a default goes against each other. I suggest you pick the latter.